OracleOpenJDK/README.md

```diff
@@ -5,10 +5,10 @@ This repository contains sample Docker configurations to facilitate installation
 ## Building the Oracle Java OpenJDK base image
 Navigate to the folder containing the Dockerfile and run docker build, tagging the image with the version number:
```

Comment:
Wrap "Dockerfile" in backtick quotes

Reply:
I think you meant "docker build" should be in backtick quotes.  If so done.